pkg/rpc/rpc.go

```diff
@@ -0,0 +1,148 @@
+package rpc
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/icinga/icinga-notifications/pkg/plugin"
+	"go.uber.org/zap"
+	"io"
+	"sync"
+)
+
+type Error struct {
+	cause error
+}
+
+func (err *Error) Error() string {
+	return fmt.Sprintf("RPCError: %s", err.cause.Error())
+}
```

Comment:
References an outdated type name in the error message, I'd go with just `"RPC error: %s"`.